tests/containers/search/filters/keyword/KeywordContainer-test.jsx

```diff
@@ -40,39 +40,12 @@ describe('KeywordContainer', () => {
             expect(toggleKeywordSpy.callCount).toEqual(1);
             expect(mockReduxAction).toHaveBeenCalled();
 
```

Comment:
I mean, we should still have a test that covers `toggleKeyword` - in this case all that method does is to trigger a Redux action (which should be what you are testing).